### ISSUE TITLE
Weekly scrum - [2020-03-19]

### ISSUE THREAD

**Issue by JoseFrancoJimenez** [JoseFrancoJimenez/miniShopify#89]
1- what have I done this week? (with link(s) to the relevant GitHub Issue(s))
2- what will I do next week? (with link(s) to the relevant GitHub Issue(s))
3- what is holding me back? (if applicable)

**Comment by JoseFrancoJimenez** [comment]
1- what have I done this week?
I have created, worked on and resolved the following issues: https://github.com/JoseFrancoJimenez/miniShopify/issues/75, https://github.com/JoseFrancoJimenez/miniShopify/issues/87,
https://github.com/JoseFrancoJimenez/miniShopify/issues/60,
https://github.com/JoseFrancoJimenez/miniShopify/issues/82,
https://github.com/JoseFrancoJimenez/miniShopify/issues/84,
https://github.com/JoseFrancoJimenez/miniShopify/issues/73.

All those issues are related to both, functionality and "look and feel". 

2- what will I do next week?
Something that we tried to do this week was to start using jhipster on our app. But it seems that the configuration if a bit complicated specially when deploying to Heroku and when using travis. I'll give it another try next week. 

However, I think that finishing the project has a greater priority than using jhipster (jhipster is only required for the presentation, not for the project). So if things go wrong with jhipster I'll just finish whatever is left on the project.

3.- what is holding me back?
This time has been harder, due to all things that are going on out there. It has been complicated to work as a team (due to the availability of every individual).  It has been specially hard for me to work since I've been sick for two weeks already, but still I am trying to do my best here.

**Comment by Aryanrt** [comment]
1- what have I done this week?
I have created, worked on and resolved the following issues: 53,#55,#67 ,#70,#77, #65, #57 

All those issues are related to both, functionality mainly but also includes Views and Models.

2- what will I do next week?
Working on moving the front end to angular and hopefully learning more about JHipster


3.- what is holding me back?
Cancelling the classes have created more assignments and reports for students since this is the only way of assessment with no classes. I spent a lot of time on JHipster and I could not get it to work.

**Comment by almasrur** [comment]
1- what have I done this week?
I have created, worked on and resolved the following issues: #91,#92,

All those issues are related to both, verification and validation mainly but also includes Documentation

2- what will I do next week?
Working on moving the front end to angular and hopefully learning more about JHipster, and creating tests for all controller classes

3.- what is holding me back?
Cancelling the classes have created more assignments and reports for students since this is the only way of assessment with no classes. I spent a lot of time on JHipster and I could not get it to work.
